src/main/java/model/Student.java

```diff
@@ -0,0 +1,83 @@
+package model;
+
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Student {
+
+    String name;
+
+    List<LocalDateTime> timeRecords;
+
+    public Student(String name, List<LocalDateTime> timeRecords) {
+        this.name = name;
+        this.timeRecords = timeRecords;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public ArrayList<LocalDateTime> getTimeRecords() {
+        return (ArrayList<LocalDateTime>) timeRecords;
+    }
+
+    public void addTime(LocalDateTime localDateTime) {
+        timeRecords.add(localDateTime);
+        Collections.sort(timeRecords);
+    }
+
+    public LocalDateTime findSameDay(LocalDateTime wantToFindLocalDateTime) {
+        for (LocalDateTime localDateTime : timeRecords) {
+            LocalDateTime dayDate1 = localDateTime.truncatedTo(ChronoUnit.DAYS);
+            LocalDateTime dayDate2 = wantToFindLocalDateTime.truncatedTo(ChronoUnit.DAYS);
+            if (dayDate1.compareTo(dayDate2) == 0) {
+                return localDateTime;
+            }
+        }
+        return null;
+    }
+
+    public void modifyRecord(LocalDateTime wantToModifyLocalDateTime) {
+        timeRecords.remove(findSameDay(wantToModifyLocalDateTime));
+        addTime(wantToModifyLocalDateTime);
+        Collections.sort(timeRecords);
+    }
+
+    public boolean isExistSameDay(LocalDateTime localDateTime1, LocalDateTime localDateTime2) {
+        LocalDateTime dayDate1 = localDateTime1.truncatedTo(ChronoUnit.DAYS);
+        LocalDateTime dayDate2 = localDateTime2.truncatedTo(ChronoUnit.DAYS);
+
+        return (dayDate1.compareTo(dayDate2) == 0);
```

Comment:
LocalDateTime 은 `isEquals` 메서드도 제공하고 있군요 :)

```suggestion
        return dayDate1.isEquals(dayDate2);
```

Reply:
해당 메서드 사용으로 수정하였습니다!